java/test/jmri/server/json/consist/JsonConsistManagerTest.java

```diff
@@ -31,6 +31,9 @@ public void testDelConsist(){
     public void setUp() {
         JUnitUtil.setUp();
         cm = new JsonConsistManager();
+        if (! ((JsonConsistManager)cm).isConsistManager()) {
+            ignoreTests = true;
```

Comment:
If the tests that this causes to not run are always ignored for this case, I could include an override of the test with an ignore ( like there is for the failing NCE cases ) instead of adding this variable here.  I think that provides better documentation than this flag does.

Reply:
I have done that change now.